gateway-service/src/main/java/com/ca/mfaas/gateway/ws/WebSocketProxyServerHandler.java

```diff
@@ -120,7 +125,7 @@ private void openWebSocketConnection(RoutedService service, ServiceInstance serv
             WebSocketRoutedSession session = new WebSocketRoutedSession(webSocketSession, targetUrl, jettySslContextFactory);
             routedSessions.put(webSocketSession.getId(), session);
         } catch (WebSocketProxyError e) {
-            log.error("Error opening WebSocket connection to {}: {}", targetUrl, e.getMessage(), e);
+            apimlLog.log("apiml.gateway.websocketConnectionError",targetUrl,e.getMessage());
```

Comment:
Note: It logs also for 404 case which is not good to log. Should be considered.

Reply:
I am for hiding it, as the errors should be handled as part of the wss connection, as it is for the http protocol, in other words the client should be notified with response and not logging error.